packages/mobile/src/firebase/dynamicLinks.ts

```diff
@@ -0,0 +1,36 @@
+import firebase from 'react-native-firebase'
+import Logger from 'src/utils/Logger'
+
+const TAG = 'firebase/dynamicLink'
+
+export async function generateShortInviteLink({
+  link,
+  playStoreUrl,
+  appStoreUrl,
+  bundleId,
+}: {
+  link: string
+  playStoreUrl: string
+  appStoreUrl: string
+  bundleId: string
+}): Promise<string> {
+  try {
+    const firebaseLink = new firebase.links.DynamicLink(link, 'https://celol.page.link')
+
+    firebaseLink.android.setFallbackUrl(playStoreUrl)
+    firebaseLink.android.setPackageName(bundleId)
+
+    firebaseLink.ios.setFallbackUrl(appStoreUrl)
+    firebaseLink.ios.setBundleId(bundleId)
+
+    // Please note other parameter than UNGUESSABLE is unsafe
+    const shortUrl = await firebase.links().createShortDynamicLink(firebaseLink, 'UNGUESSABLE')
+
+    // It is NOT recommended to shorten this link because it
+    // can break deep links
```

Comment:
Do you have more info on this?

Reply:
I've got the hint from @martinvol that using other shortener (like bit.ly) on on top of already shortened link is a bad idea. I did not investigate it deeply why, but we do not have that many reasons to use (text now fits in a SMS just fine). Comment was left by @martinvol.

Reply:
Ok, I thought it was related to an option in the dynamic link generation where it could output a shorter link.